public/datasources.json

```diff
@@ -1,4 +1,5 @@
 [
+    {"name": "Arab Bank Australia Limited", "url": "https://openbanking-api.arabbank.com.au/public/cds-au/v1", "icon": "https://www.arabbank.com.au/themes/arabbank/images/abal-logo.svg"}
```

Comment:
Missing `,` at the end of the closing brace.

Reply:
Comma has been added, please review and approve if no issues.

Thanks,
Jay